src/App.tsx

```diff
@@ -1,95 +1,129 @@
 import React from 'react';
 import 'bulma/css/bulma.css';
 import './App.scss';
+import classNames from 'classnames';
+import goodsFromServer from './server/goodsFromServer.json';
+import { GoodList } from './components/GoodList';
 
-export const goodsFromServer = [
-  'Dumplings',
-  'Carrot',
-  'Eggs',
-  'Ice cream',
-  'Apple',
-  'Bread',
-  'Fish',
-  'Honey',
-  'Jam',
-  'Garlic',
-];
-
-enum SortType {
-  NONE,
-  ALPHABET,
-  LENGTH,
-}
+type State = {
+  sortType: string;
+  isReversed: boolean;
+  isResetActive: boolean;
+};
 
 type ReorderOptions = {
-  sortType: SortType,
-  isReversed: boolean,
+  sortType: string;
+  isReversed: boolean;
 };
 
-// Use this function in the render method to prepare goods
 export function getReorderedGoods(
   goods: string[],
   { sortType, isReversed }: ReorderOptions,
 ) {
-  // To avoid the original array mutation
   const visibleGoods = [...goods];
 
-  // Sort and reverse goods if needed
-  // eslint-disable-next-line no-console
-  console.log(sortType, isReversed);
+  visibleGoods.sort((g1, g2) => {
+    switch (sortType) {
+      case 'ALPHABET':
```

Comment:
It would be great to create Enum with all possible sortTypes, so you don't need to hardcore them